src/cdk/generator.ts

```diff
@@ -131,9 +132,14 @@ export class ResourceGenerator extends Construct {
     // emit CDK constructs for specified resources
     resources.forEach((resource) => this.generateConstructsForResource(resource))
 
+    const namePostfix = randomBetweenNAndM(1000, 2000)
```

Comment:
Please remove the random number

Reply:
You can remove `exportName` if we don't have httpApiName
Honestly we don't really need `exportName` at all it's just a nice convenience but really not worth this headache

Reply:
yeah I agree, the exportName is should be optional but the problem comes from the resource ID the `CfnOutput` has an ID and also it has to be unique, so we depend on either the `this.httpApi.httpApiName` or we depend on a random number / sequence or something to make it unique, as log as `this.httpApi.httpApiName` is not null we are not going to have a problem, so I deleted the function for the random number